### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by hellohien**
Screen Shot 2021-03-30 at 11 26 19 AM
Screen Shot 2021-03-30 at 11 27 08 AM

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -1,8 +1,35 @@
 /* exported data */
+var $form = document.getElementById('form');
+var $photoUrl = document.getElementById('photo-url');
+var $title = document.getElementById('title');
+var $notes = document.getElementById('notes');
```
Comment (codingcodymiller) [inline js/data.js]: This should be in `main.js`

File: js/data.js [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+
+  data.nextEntryId++;
+  data.entries.unshift(entries);
+  $form.reset();
+
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('javascript-local-storage', dataJSON);
+}
+
+$form.addEventListener('submit', addEntry);
```
Comment (codingcodymiller) [inline js/data.js]: This should be in `main.js`

File: js/data.js [hunk 3]
```diff
@@ -1,8 +1,35 @@
...
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+var previousDataJSON = localStorage.getItem('javascript-local-storage');
+if (previousDataJSON) {
+  data = JSON.parse(previousDataJSON);
+}
+
```
Comment (codingcodymiller) [inline js/data.js:13]: Do you have a `beforeunload` event handler? I don't see it 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hellohien** [comment]
Screen Shot 2021-03-30 at 12 31 07 PM
Screen Shot 2021-03-30 at 12 31 00 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**